src/components/resource-picker/wikimedia-search.yml

```diff
@@ -14,8 +14,8 @@ searchFileType_pdf:
   en: PDF
   de: PDF
 searchInvitationHeader:
-  en: Search in the Wikimedia Commons Database
-  de: Suche in der Wikimedia-Commons-Datenbank
+  en: Search in the Wikimedia Database
```

Comment:
As we're on it, I think it would be better to lower-case the word "database" as it is not part of the official name.

Reply:
Oh of course, doesn't work in English. Changed.